### PR TITLE
Updated 91_version_history file for 2.4.0 release

### PR DESCRIPTION
**Pull request by Mohit04tomar**

### CODE REVIEW

File: docs/91_version_history.rst [hunk 1]
```diff
@@ -34,35 +33,93 @@ Release Date: 2024-04-10
...
+       - An issue that could cause excessive messages regarding connection loss and regain to be raised in the OMF north plugin has been resolved.
+       - An issue that caused the fledge-north-kafka (C based) plugin to fetch data when it was disabled has been resolved.
+       - If you set the User Authentication Policy to username, you must select a Security Policy other than None to communicate with the OPC UA Server. Allowing username authentication with None would mean that usernames and passwords would be passed from the plugin to the server as clear text which is a serious security risk. This is explained in the `OPC UA Specification <opcua-spec>`_. In addition, OPC UA defines a Security Policy for a "UserIdentityToken". When configuring the fledge-south-s2opcua plugin, the Security Policy selected in your configuration must match a supported "UserIdentityToken" Security Policy.  To help troubleshoot configuration problems, log messages for the endpoint search have been improved. The documentation includes a new section called "Username Authentication".
```
Comment (ashish-jabble) [inline docs/91_version_history.rst:115]: What is `<opcua-spec>`? Provide the relative link

File: docs/91_version_history.rst [hunk 2]
```diff
@@ -34,35 +33,93 @@ Release Date: 2024-04-10
...
 - **GUI**
 
     - New Features:
 
-       - 
+       - A new page has been added for managing additional services within an instance.
+       - Support for entering simple lists for configuration items has been added.
+       - Support has been added for manipulating key/value lists using the new available list configuration type that is available.
+       - Navigation buttons have been added to the tabs in the south and north services to facilitate easier navigation between tabs.
+       - A preview of the new flow editor for the north side has been added. This may be enabled via the GUI settings page.
```
Comment (ashish-jabble) [inline docs/91_version_history.rst:78]: only at North side?

Comment (MarkRiddoch) [reply docs/91_version_history.rst:78]: South was in 2.3.0

**Review by MarkRiddoch — APPROVED** [review]

File: docs/91_version_history.rst (hunk 2 above)
Comment (ashish-jabble) [reply docs/91_version_history.rst:78]: Flow editor feature in `2.3.0_next`

**Pull request merged**